### ISSUE TITLE
crash with litho 0.28.0

### ISSUE THREAD

**Issue by ironmanTony** [pavlospt/litho-glide#7]
```
Caused by: java.lang.NoSuchMethodError: No super method release()V in class Lcom/facebook/litho/Component$Builder; or its super classes (declaration of 'com.me.demo.Component$Builder' appears in /data/app/com.me.demo-A4WIT__Od95fwsLcSBiVsg==/base.apk!classes2.dex)
        at com.github.pavlospt.litho.glide.GlideImage$Builder.release(GlideImage.java:504)
        at com.github.pavlospt.litho.glide.GlideImage$Builder.build(GlideImage.java:498)
        at com.github.pavlospt.litho.glide.GlideImage$Builder.build(GlideImage.java:323)
```

**Comment by pavlospt** [comment]
Will update the dependencies once I have some free time! Feel free to submit a PR if you need it earlier :)

**Comment by pavlospt** [comment]
I updated Litho and migrated to AndroidX as well! It should be fixed in version 1.6 which will be available in a few hours! If the problem persists, please submit a reproducing project!
